blocks/init/src/blocks/components/header/header-style.scss

```diff
@@ -33,26 +33,34 @@ $header: (
   }
 
   &__column {
-    flex: 1 0 auto;
-    height: 100%;
+    display: flex;
+    flex: 1 0 1px;
```

Comment:
Why `flex-basis: 1px`?

Reply:
I've mentioned in the changelog:

>Fixed a bug with centering where logo wouldn't be quite centered on mobile - seemingly due to having flex-basis set to auto. Setting it to any other value (such as 1px) fixed the issue.

Not sure why this is happening tho.

Most likely specific implementations of these 3 flex columns will have to be tweaked for your project. (for example, as I've noticed while doing our Gutenberg task, it's behaving kinda weird when you have 2 items in a single column - when `flex-basis: 1px` - but it works much better for the default header this way).